recipes/r-srvyr/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set version = '1.2.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-srvyr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/srvyr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/srvyr/srvyr_{{ version }}.tar.gz
+  sha256: 2d759fab2527b329c30534b98c18503a3f62462dc9aee0882bc42dcf642c7e30
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-dplyr >=1.1.0
+    - r-magrittr
+    - r-rlang
+    - r-survey >=4.1
+    - r-tibble
+    - r-tidyr
+    - r-tidyselect
+    - r-vctrs >=0.3.0
+  run:
+    - r-base
+    - r-dplyr >=1.1.0
+    - r-magrittr
+    - r-rlang
+    - r-survey >=4.1
+    - r-tibble
+    - r-tidyr
+    - r-tidyselect
+    - r-vctrs >=0.3.0
+
+test:
+  commands:
+    - $R -e "library('srvyr')"           # [not win]
+    - "\"%R%\" -e \"library('srvyr')\""  # [win]
+
+about:
+  home: http://gdfe.co/srvyr/, https://github.com/gergness/srvyr/
+  license: GPL-2.0-or-later
+  summary: Use piping, verbs like 'group_by' and 'summarize', and other 'dplyr' inspired syntactic
+    style when calculating summary statistics on survey data using functions from the
+    'survey' package.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - vitodelgrec
```

We suggest including the Conda Forge R Team:

```suggestion
    - conda-forge/r
    - vitodelgrec
```